socorro/cron/jobs/bugzilla.py

```diff
@@ -155,10 +145,14 @@ def inner_transaction(
         if not useful:
             raise NothingUsefulHappened('nothing useful done')
 
-    def _iterator(self, query):
-        for report in csv.DictReader(urllib2.urlopen(query)):
+    def _iterator(self, from_date):
+        payload = BUGZILLA_PARAMS.copy()
+        payload['chfieldfrom'] = from_date
+        r = requests.get(BUGZILLA_BASE_URL, params=payload)
+        results = r.json()
```

Comment:
One thing we've often seen is that bugzilla's rest API simply fails. And when it does, the error message is in HTML. 
It's more of a nit but it might be worth doing an assertion of sorts on the `r.status_code` first. Otherwise you get a nasty JSONDecodeError which might be confusing. 
Either `assert r.status_code == 200, r.status_code` or `if r.status_code != 200: return`